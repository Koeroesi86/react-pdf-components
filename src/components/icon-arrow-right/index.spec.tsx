import React from "react";
import { renderToString } from "@react-pdf/renderer";
import { Wrapper } from "@test-helpers";
import IconArrowRight from "./index";

describe("<IconArrowRight/>", () => {
  it("should render", async () => {
    const result = await renderToString(
      <Wrapper>
        <IconArrowRight width={12} height={12} color="#000"/>
      </Wrapper>
    );
    expect(result).toMatchSnapshot();
  });
});
