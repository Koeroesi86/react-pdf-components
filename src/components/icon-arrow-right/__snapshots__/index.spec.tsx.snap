// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconArrowRight/> should render 1`] = `
"%PDF-1.3
%����
7 0 obj
<<
/Type /Page
/Parent 1 0 R
/MediaBox [0 0 595.280029 841.890015]
/Contents 5 0 R
/Resources 6 0 R
>>
endobj
6 0 obj
<<
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI]
>>
endobj
5 0 obj
<<
/Length 602
>>
stream
1 0 0 -1 0 841.890015 cm
q
q
1 0 0 1 0 0 cm
q
q
0 3 m
12 3 l
12 3 12 3 12 3 c
12 15 l
12 15 12 15 12 15 c
0 15 l
0 15 0 15 0 15 c
0 3 l
0 3 0 3 0 3 c
h
W n
1 0 0 1 0 3 cm
0.023438 0 0 0.023438 0 0 cm
1 0 0 1 96 0 cm
q
/DeviceRGB cs
0 0 0 scn
285.476 272.971 m
91.132 467.314 l
81.759 476.687 66.563 476.687 57.191 467.314 c
34.524 444.647 l
25.167 435.29 25.149 420.125 34.484 410.746 c
188.505 256 l
34.484 101.255 l
25.149 91.876 25.167 76.711 34.524 67.354 c
57.191 44.687 l
66.564 35.314 81.76 35.314 91.132 44.687 c
285.475 239.03 l
294.848 248.402 294.848 263.598 285.476 272.971 c
h
f
Q
Q
Q
Q
Q

endstream
endobj
9 0 obj
(react-pdf)
endobj
10 0 obj
(react-pdf)
endobj
11 0 obj
(D:20210414183756Z)
endobj
8 0 obj
<<
/Producer 9 0 R
/Creator 10 0 R
/CreationDate 11 0 R
>>
endobj
3 0 obj
<<
/Type /Catalog
/Pages 1 0 R
/Names 2 0 R
/ViewerPreferences 4 0 R
>>
endobj
1 0 obj
<<
/Type /Pages
/Count 1
/Kids [7 0 R]
>>
endobj
2 0 obj
<<
/Dests <<
  /Names [
]
>>
>>
endobj
4 0 obj
<<
/DisplayDocTitle true
>>
endobj
xref
0 12
0000000000 65535 f 
0000001105 00000 n 
0000001162 00000 n 
0000001018 00000 n 
0000001209 00000 n 
0000000200 00000 n 
0000000133 00000 n 
0000000015 00000 n 
0000000944 00000 n 
0000000853 00000 n 
0000000880 00000 n 
0000000908 00000 n 
trailer
<<
/Size 12
/Root 3 0 R
/Info 8 0 R
>>
startxref
1252
%%EOF
"
`;
